File: src/mysqlcookbook/recipes/tblmgmt/uniq_name.pl
#!/usr/bin/perl
# uniq_name.pl: show how to use PID to create table name

use strict;
use warnings;

print "PID: $$\n";
#@ _GENERATE_NAME_WITH_PID_
my $tbl_name = "tmp_tbl_$$";
#@ _GENERATE_NAME_WITH_PID_
print "Table name: $tbl_name\n";

File: src/mysqlcookbook/recipes/tblmgmt/uniq_name.php
<?php
# uniq_name.php: show how to use PID to create table name

#@ _GENERATE_NAME_WITH_PID_
$tbl_name = "tmp_tbl_" . posix_getpid ();
#@ _GENERATE_NAME_WITH_PID_
print ("$tbl_name\n");
?>
